website/src/components/subscriptions/create/ChannelsInput.tsx

```diff
@@ -0,0 +1,61 @@
+import type { Channel } from '../../../types/NotificationChannels.ts';
+import { useEffect, useState } from 'react';
+import { DividerList } from '../../../styles/containers/DividerList.tsx';
+
+export function ChannelsInput({
+    label,
+    channels,
+    onChannelsSelect,
+}: {
+    label: string;
+    channels: Channel[];
+    onChannelsSelect: (value: string[]) => void;
+}) {
+    const [selectedChannels, setSelectedChannels] = useState<string[]>([]);
+
+    useEffect(() => {
+        onChannelsSelect(selectedChannels);
+    }, [selectedChannels]);
```

Comment:
We still need it. We don't want to run the function `onChannelSelect`, while the component rerenders, but after that. Otherwise, we get those funny rendering issues, where the state is only partial updated, or not at all.
(I tried to create InputComponents, which manage their own state and the calling side only gets the changed property, similar to `<input>`).